returnn/tf/layers/basic.py

```diff
@@ -12,7 +12,7 @@
 import returnn.tf.util.basic as tf_util
 from returnn.util.basic import unicode, NotSpecified
 from returnn.tf.util.data import Data, SearchBeam
-from returnn.tf.util.basic import OutputWithActivation, dimshuffle, swapaxes
+from returnn.tf.util.basic import OutputWithActivation, dimshuffle, swapaxes, pad_replicate
```

Comment:
This is not needed (use `tf_util`), and in general, we prefer to not import too much things.
```suggestion
from returnn.tf.util.basic import OutputWithActivation, dimshuffle, swapaxes
```